css/main.css

```diff
@@ -1,11 +1,6 @@
-body {
-	padding-top: 3em;
-	max-width: 45em;
```

Comment:
What padding do you suggest then ? I believe 1 or 1.5em  is a good compromise between 0 and 3.